lib/pf/password.pm

```diff
@@ -140,11 +140,16 @@ Generates the password
 =cut
 
 sub _generate_password {
+    my ($size) = @_;
+    my $min = 8;
+    my $max = 12;
 
-    my $password = word(8, 12);
+    $min = $max = $size if (defined $size);
```

Comment:
This instruction doesn't limit size of `password` to 12 characters. With following one-liner:

```shell
perl -I/usr/local/pf/lib -Mpf::password -e "pf::password::generate("potd0",[{type => 'valid_from', value => '2018-08-27 09:39:05'},{type => 'expiration', value => '2018-08-27 09:50:05'}],undef, 0, {'password_length' => 64})"
```

`$size` equals to `64`

Reply:
Moreover, I didn't see `Password Length` limitation when configuring source.

Reply:
I am not sure to understand why do we want to limit the password length to 12 ?

Reply:
On my side, I'm able to freeze PF (with a little config) when setting a password of 64 characters.

Reply:
It's because of Crypt::GeneratePassword::word who is taking time to generate a readable password.
I limited the password length from 1 to 15.